app/webpacker/components/Tickets/api/getComments.js

```diff
@@ -0,0 +1,9 @@
+import { fetchJsonOrError } from '../../../lib/requests/fetchWithAuthenticityToken';
+import { viewUrls } from '../../../lib/requests/routes.js.erb';
+
+export default async function getComments({ ticketId }) {
+  const { data } = await fetchJsonOrError(
+    viewUrls.tickets.listComments(ticketId),
+  );
+  return data?.comments || [];
```

Comment:
Not a big fan of already accessing the data here. Do we do this in other places throughout the code?